### PR TITLE
chore: [CHAOS-2381]: Support HTTPS with authentication in http probe

### PR DESCRIPTION
**Pull request by ispeakc0de**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Comment by ravilach** [comment]
@ispeakc0de similar feedback from other PR also. Can you check and escape please? Can run a build locally on your machine also with `yarn build`. Thanks!

```
SyntaxError: /harness/docs/chaos-engineering/technical-reference/probes/http-probe.md: Expected corresponding JSX closing tag for <td>. (372:2)

  370 |    </td>

  371 |    <td>The <code>credentials</code> consists of file path for basic authentication credentials or a bearer token, which are then attached to the experiment pod as volume secrets. These secret resources contain either the username:password encoded in base64 format or a bearer token, depending on the authentication type

> 372 |   </tr>

      |   ^

  373 | </table>

  374 | <h3 {...{"id":"tls"}}>{`TLS`}</h3>

  375 | <p>{`It offers the mechanism to validate TLS certifications for the http endpoint. You can supply the 'cacert,' or the client certificate and client key, to perform the validation.

[ERROR] Client bundle compiled with errors therefore further build is impossible
```

File: docs/chaos-engineering/technical-reference/probes/http-probe.md [hunk 1]
```diff
@@ -337,6 +337,127 @@ Listed below is the probe schema for HTTP Probe with common properties shared ac
   </tr>
 </table>
 
+### Authentication
+
+This establishes a fundamental authentication mechanism for the http endpoint. The username:password, encoded in base64 or bearer token should be placed either within the 'credentials' field or as a file path in the 'credentialsFile' field.
```
Comment (KimberlyFields) [inline docs/chaos-engineering/technical-reference/probes/http-probe.md:342]:
```suggestion
This establishes a fundamental authentication mechanism for the http endpoint. The username:password, encoded in base64 or bearer token should be placed either within the `credentials` field or as a file path in the `credentialsFile` field.
```

File: docs/chaos-engineering/technical-reference/probes/http-probe.md [hunk 2]
```diff
@@ -337,6 +337,127 @@ Listed below is the probe schema for HTTP Probe with common properties shared ac
...
+It's important to note that `credentials` and `credentialsFile` are two options that cannot be used simultaneously
```
Comment (KimberlyFields) [inline docs/chaos-engineering/technical-reference/probes/http-probe.md:343]:
```suggestion
It's important to note that `credentials` and `credentialsFile` are two options that cannot be used simultaneously.
```

File: docs/chaos-engineering/technical-reference/probes/http-probe.md [hunk 3]
```diff
@@ -337,6 +337,127 @@ Listed below is the probe schema for HTTP Probe with common properties shared ac
...
+   </td>
+   <td><code>string</code>
+   </td>
+   <td>The <code>credentials</code> consists of file path for basic authentication credentials or a bearer token, which are then attached to the experiment pod as volume secrets. These secret resources contain either the username:password encoded in base64 format or a bearer token, depending on the authentication type
+  </tr>
+</table>
+
+### TLS
+
+It offers the mechanism to validate TLS certifications for the http endpoint. You can supply the 'cacert,' or the client certificate and client key, to perform the validation.
```
Comment (KimberlyFields) [inline docs/chaos-engineering/technical-reference/probes/http-probe.md:397]:
```suggestion
It offers the mechanism to validate TLS certifications for the http endpoint. You can supply the `cacert` or the client certificate and client key to perform the validation.
```

File: docs/chaos-engineering/technical-reference/probes/http-probe.md [hunk 4]
```diff
@@ -337,6 +337,127 @@ Listed below is the probe schema for HTTP Probe with common properties shared ac
...
+Alternatively, you have the option to enable the 'insecureSkipVerify' check to bypass certificate validation
```
Comment (KimberlyFields) [inline docs/chaos-engineering/technical-reference/probes/http-probe.md:398]:
```suggestion
Alternatively, you have the option to enable the `insecureSkipVerify` check to bypass certificate validation.
```

**Review by KimberlyFields — APPROVED** [review]

**Pull request merged**